src/components/pages/Photos/AboutBlock.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import './PhotoPage.css';
+
+export default class AboutBlock extends Component {

```

Comment:
I think `AboutRover` would be a more descriptive name for this class, what do you think?